cmake/libappimageConfig.cmake.in

```diff
@@ -1,7 +1,11 @@
 # - Config file for the AppImage package
-# It defines the following variables
+# Exported Variables
```

Comment:
Can we please add a comment that using these variables is discouraged and they might be dropped in the future?

Reply:
@azubieta can you please add said comment? Then we can merge this and @kossebau can continue on KDE Phab.

Reply:
Done